ciw/node.py

```diff
@@ -144,7 +150,13 @@ def begin_service_if_possible_release(self, current_time):
         """
         if self.free_server() and self.c != float('Inf'):
             srvr = self.find_free_server()
-            if len([i for i in self.all_individuals if not i.server]) > 0:
+            if len(self.interrupted_individuals) > 0:
```

Comment:
This looks like its the same code as lines 133-139, could it be wrapped in a function/method?